Jenkinsfile

```diff
@@ -26,32 +30,41 @@ node("k8s") {
             // anything to include it anyway.
             withRepoKey {
                 k8s_contexts.each { cluster ->
-                    template(
-                        basedir: "${env.WORKSPACE}/${cluster}",
-                        cluster: cluster
-                    )
-                    sh("ls ${env.WORKSPACE}/${cluster}/tmp-k8s/")                 
+                    template(cluster: cluster)
+                    sh("kubectl --context ${cluster} apply -f ./tmp-k8s")                 
                 }
             }
         }
     }
 
-    if (env.BRANCH_NAME.equals("main")) {
-        stage("Deploy") {
-            k8s_contexts = [
-                "staging",
-            ]
+    else if (env.BRANCH_NAME.equals("staging")) {
```

Comment:
This jenkinsfile will not even apply the proper specs with our current directory structure. We only want one "Deploy" phase, and it should look like the first one that is currently on the main branch, except we want to add:

```
sh("kubectl --context ${cluster} apply -f ${env.WORKSPACE}/${cluster}/tmp-k8s/")
```

after the ```ls``` command. also remove the sleep infinity